Library/ViewModels/ProjectPamphletMainCellViewModel.swift

```diff
@@ -354,6 +340,51 @@ private func fundingStatus(forProject project: Project) -> String {
   }
  }
 
+private func pledgeAmountAndGoalAndCountry(forProject project: Project,
+                                           needsConversion: Bool) -> (Int, Int, Project.Country) {
```

Comment:
Would a `typealias` be useful here? Perhaps something like:

```swift
typealias PledgedAmountData = (pledged: Int, goal: Int, country: Project.Country)
```

This would improve readability when referring to the tuple members later on:
`pledgedAmountData.goal` vs `pledgedGoalCountry.1`